bluez-generated/introspect.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Bash >= 4 does not come with osx because of its GPLv3 license.
+# Install it via homebrew to get associative array support.
+
+set -euxo pipefail
+
+cd "$(dirname "$0")"
+
+if [ ${INTROSPECT:-1} = 1 ]; then
+    ssh pi@raspberrypi.local \
+        gdbus introspect --system --dest org.bluez --object-path / --recurse \
```

Comment:
- [ ] Split this out into a $GDBUS so that it can be set to 'gdbus' on localhost or 'sshpi@raspberrypi.local gdbus' for remote introspection.